dix/req.c

```diff
@@ -1977,7 +1977,7 @@ static int dix__spare_target(struct m0_dix_rec_op         *rec_op,
 		if (rc != 0)
 			return M0_ERR(rc);
 		spare = &rec_op->dgp_units[spare_offset + slot];
-		if (!spare->dpu_failed) {
+		if (!spare->dpu_unavail) {
 			/* Found non-failed spare unit, exit the loop. */
```

Comment:
[*suggestion*]Can we change the comment to say /* Found an available spare unit, ..*/

Reply:
Done.